docs/pages/desktop-access/getting-started.mdx

```diff
@@ -382,6 +382,17 @@ Right-click on `Server authentication certificate template`, `Edit`, then select
 </Figure>
 
 
+### Configure server certificate auto-enrollment
+
+In the group policy editor for `Default Domain Policy`, select:
```

Comment:
Generally we don't want to recommend editing the default domain policy.

Instead, let's put this setting in the GPO we're already having users create.